From 7c5801f42e756f477b0d4a8862088e0fa2e5d600 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 1 May 2019 10:24:40 +1000
Subject: [PATCH] MANTA-4242 Muppet fails to change haproxy config after a
 cueball/zk shuffle Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
 Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index c54fb9f..f5d072c 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "once": "1.4.0",
         "vasync": "2.2.0",
         "verror": "1.10.0",
-        "zkstream": "0.11.3"
+        "zkstream": "0.11.4"
     },
     "devDependencies": {
         "nodeunit": "0.11.2",
-- 
2.21.0

